solcjs

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+set -e
+
+script_dir="$(cd "$(dirname "$0")" && pwd)"
+
+[ -e "$script_dir/dist/solc.js" ] || npm run build
```

Comment:
shouldn't this be
```suggestion
[ -e "$script_dir/dist/solc.js" ] || (cd "$script_dir" && npm run build)
```

Reply:
or maybe fail instead of building transparently?

Reply:
Ah, true. I only tested it quickly with `--version` and missed that completely :) Fixed.

I think that building transparently is better because the whole point of this is to keep things backwards-compatible. If you can adjust your workflow to run `npm build`, you can just as well switch to `dist/solc.js`. I added a comment to make this clear.